packages/docs/docs/plugin/official/plugin-clean-urls.md

```diff
@@ -22,6 +22,12 @@ module.exports = {
 }
 ```
 
+::: warning
+VuePress **does not have the right** to modify server identification. If you want your URLs to follow a certain pattern (e.g. `/routing` instead of `/routing.html` or `routing/`), you should make sure that your server would treat it as an HTML. This mean that you may need to configure your server specifically.
+
```

Comment:
mean -> means.

Reply:
Maybe adding some refs would be better here.

https://www.netlify.com/docs/redirects/#trailing-slash
https://surge.sh/help/using-clean-urls-automatically

Reply:
Maybe writing logic should be:

> Note that plugin always work on dev, but VuePress **does not have the right** to ...